### PR TITLE
GSNGM-821: Add a link-view to a project

### PR DESCRIPTION
**Pull request by b4l**

### CODE REVIEW

File: api/src/lib.rs [hunk 1]
```diff
@@ -31,11 +34,25 @@ pub async fn app(pool: PgPool) -> Router {
...
+                        .allow_methods(vec![Method::GET, Method::POST, Method::PUT, Method::DELETE])
+                        .allow_origin(Origin::list(vec![
+                            "http://localhost:8000".parse().expect("parse origin"),
+                            "https://api.dev.swissgeol.ch"
+                                .parse()
+                                .expect("parse origin"),
+                            "https://api.int.swissgeol.ch"
+                                .parse()
+                                .expect("parse origin"),
+                            "https://api.swissgeol.ch".parse().expect("parse origin"),
```
Comment (gberaudo) [inline api/src/lib.rs:52]: You should also allow the likes of https://dev.swissgeol.ch

**Review by gberaudo — APPROVED** [review]

File: api/src/lib.rs (hunk 1 above)
Comment (b4l) [reply api/src/lib.rs:52]: Including `viewer.swissgeol.ch`, `int.swissgeol.ch`, `review.swissgeol.ch`?

Comment (b4l) [reply api/src/lib.rs]: Addressed in https://github.com/swissgeol/ngm/pull/855/commits/881d27ec98c5548d115a3cc5f42e2dbc4142122f

**Comment by b4l** [comment]
@fredj or @gberaudo there is a bug, when creating a project, selecting a view, deselect/do some other stuff, select project view again, the view title does not display in the popup.

**Comment by fredj** [comment]
> @fredj or @gberaudo there is a bug, when creating a project, selecting a view, deselect/do some other stuff, select project view again, the view title does not display in the popup.

fixed

**Pull request merged**